src/Common/src/System/PasteArguments.cs

```diff
@@ -17,8 +17,9 @@ public static string Paste(IEnumerable<string> arguments, bool pasteFirstArgumen
         {
             var stringBuilder = new StringBuilder();
 
-            foreach (string argument in arguments)
```

Comment:
Why is the class/method named "paste"? Seems like a misnomer to me, and it should be "format" or something like that. This has nothing to do with ctrl-v.